requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
Remind me why we are moving from known working versions of these modules to installing the latest? It's not uncommon for a breaking change to be released in Docker at which point we may be forced to pin the version docker-compose.